### DISCUSSION TITLE
Browser versions statistics

### DISCUSSION THREAD

**Discussion by iamoskvin** [apify/fingerprint-suite#325] [General]
For me, it looks like header generator produces very outdated Chrome versions. It's statistics are probably more than 1 month old. 
What is the most popular Chrome version right now on desktop? The header generator produces Chrome 124. 
We could look for example at this stat.
https://www.w3counter.com/globalstats.php
https://www.stetic.com/market-share/browser/
Any thoughts?

**Comment by 0xARYA** [comment]
https://github.com/apify/fingerprint-suite/discussions/308#discussioncomment-10266392

The comment above from one of the contributors mentions how their dataset is indicative of current adoption rates, should see it better reflect new versions within the next few data updates.

**Reply by iamoskvin** [reply]
> [#308 (comment)](https://github.com/apify/fingerprint-suite/discussions/308#discussioncomment-10266392)
> 
> The comment above from one of the contributors mentions how their dataset is indicative of current adoption rates, should see it better reflect new versions within the next few data updates.

Thanks, I read it.
But for me, it looks like:
1) OR I really missed something.
2) OR The dataset works very badly or is broken, and the team needs to look into it.

I don't think that right now the browsers frequencies are much usable. Also, I am thinking about manually choosing Chrome versions, but I need to get a better dataset for this also. I would appreciate any help with this.
@barjin

**Reply by 0xARYA** [reply]
> > [#308 (comment)](https://github.com/apify/fingerprint-suite/discussions/308#discussioncomment-10266392)
> 
> > 
> 
> > The comment above from one of the contributors mentions how their dataset is indicative of current adoption rates, should see it better reflect new versions within the next few data updates.
> 
> 
> 
> Thanks, I read it.
> 
> But for me, it looks like:
> 
> 1) OR I really missed something.
> 
> 2) OR The dataset works very badly or is broken, and the team needs to look into it.
> 
> 
> 
> I don't think that right now the browsers frequencies are much usable. Also, I am thinking about manually choosing Chrome versions, but I need to get a better dataset for this also. I would appreciate any help with this.
> 
> @barjin 

https://gs.statcounter.com/browser-version-market-share

I can try to compare the browser version distribution of the package to the statistics above later tonight. We'll get some sort of hard numbers on the deviation.

**Reply by iamoskvin** [reply]
> > > [#308 (comment)](https://github.com/apify/fingerprint-suite/discussions/308#discussioncomment-10266392)
> > 
> > 
> > > 
> > 
> > 
> > > The comment above from one of the contributors mentions how their dataset is indicative of current adoption rates, should see it better reflect new versions within the next few data updates.
> > 
> > 
> > Thanks, I read it.
> > But for me, it looks like:
> > 
> > 1. OR I really missed something.
> > 2. OR The dataset works very badly or is broken, and the team needs to look into it.
> > 
> > I don't think that right now the browsers frequencies are much usable. Also, I am thinking about manually choosing Chrome versions, but I need to get a better dataset for this also. I would appreciate any help with this.
> > @barjin
> 
> https://gs.statcounter.com/browser-version-market-share
> 
> I can try to compare the browser version distribution of the package to the statistics above later tonight. We'll get some sort of hard numbers on the deviation.

Thanks, sounds good. Any results?
